status/deprecated/5/secure-transport.md

```diff
@@ -0,0 +1,239 @@
+---
+title: SECURE-TRANSPORT
+name: Secure Transport
+status: draft
+description: This document describes how Status provides a secure channel between two peers, providing confidentiality, integrity, authenticity, and forward secrecy.
+editor: Filip Dimitrijevic <filip@status.im>
+contributors:
+  - Andrea Maria Piana <andreap@status.im>
+  - Corey Petty <corey@status.im>
+  - Dean Eigenmann <dean@status.im>
+  - Oskar Thorén <oskar@status.im>
+  - Pedro Pombeiro <pedro@status.im>
+---
+
+## Abstract
+
+This document describes how Status provides a secure channel between two peers,
+providing confidentiality, integrity, authenticity, and forward secrecy.  
+It is transport-agnostic and works over asynchronous networks.
+
+It builds on the X3DH and Double Ratchet specifications,  
+with adaptations to operate in a decentralized environment.
+
+## Introduction
+
+This document describes how nodes establish a secure channel  
+and how various conversational security properties are achieved.
+
+## Definitions
+
+Perfect Forward Secrecy provides assurances that session keys remain secure  
+even if private keys are later compromised.  
+Secret channel describes a communication channel  
+where the Double Ratchet algorithm is in use.
+
+## Design Requirements
+
+- **Confidentiality**: The adversary should not be able to learn what data  
+  is being exchanged between two Status clients.
+- **Authenticity**: The adversary should not cause either endpoint  
+  of a Status 1:1 chat to accept data from any third party  
+  as though it came from the other endpoint.
+- **Forward Secrecy**: The adversary should not learn what data was exchanged  
+  between two Status clients, even if later the adversary compromises  
+  one or both of the endpoint devices.
+- **Integrity**: The adversary should not cause either endpoint  
+  of a Status 1:1 chat to accept tampered data.
+
+These properties are ensured by Signal’s Double Ratchet.
```

Comment:
```suggestion
These properties are ensured by [Signal’s Double Ratchet](https://signal.org/docs/specifications/doubleratchet/).
```
The links should not have been removed. Add the missing links and also add a Reference section, and include all the links in the order they appear in the specs. This needs to be done on all the specs.